internal/cli/hook.go

```
@@ -72,11 +94,60 @@ func newHookCmd() *cobra.Command {
+	cmd.Flags().StringVar(&inputFile, "input-file", "", "Cursor Windows adapter payload file")
+	_ = cmd.Flags().MarkHidden("input-file")
 	_ = cmd.MarkFlagRequired("connector")
 
 	return cmd
 }
 
+const cursorHookInputMaxBytes int64 = 1 << 20
+
+func openCursorHookInputFile(hookDir, path string) (*os.File, error) {
+	if !filepath.IsAbs(path) {
+		return nil, fmt.Errorf("Cursor hook input path must be absolute")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)